### DISCUSSION TITLE
Home Screen for Akira

### DISCUSSION THREAD

**Discussion by ghost** [akiraux/Akira#550] [Ideas]
## Quick Mock-up
![AkiraDesign](https://user-images.githubusercontent.com/77546233/114295740-7ea6ab00-9aa7-11eb-8f74-96e96ee2e50a.png)
## What is this ?
this is a home screen for the user to create a new file with automatically adding the artboard size he choose
or to view and open his recent files

## The Mock-up components
it contains 2 panels top panel and bottom panel

- Top Panel : Contains 4 more panels each panel contains an Image for the device and a drop-down for more device sizes except for the 4th it contains 2 text boxes one for the custom width and the second is for the custom height
- Bottom Panel : contains all the recent files that the user made and when right clicked-on it will show a context menu to remove,rename,open but on left click open the file

**Comment by LucileDT** [comment]
It would be great to let users configure what kind of device they want to be listed in this screen (as a user, I don't want to have a list containing all existing devices nor only have the ones the developers chose to put in the application by default).

So I'm adding those feature requests:
- show by default a list of maximum 4-5 devices
- under it, add a button or link `Change default devices` (not sure of the wording) which opens the corresponding settings page
- allow users to check/uncheck in the corresponding settings page predefined devices grouped by brand and alphabetically ordered (e.G. Apple[iphone 6/7/8, ipad], Fairphone[1, 2, 3], Samsung[s7, s8]...)
- have a search bar in the corresponding settings page to easily find a particular device
- allow users to create new brands & devices (brand name, device name, device size, icon)
- maybe add a search bar in the homepage to find a particular device without having to set it by default neither having to use the custom size option?

**Reply by ghost** [reply]
If you click on the arrow beside the device name you will get a dropdown with a list of more devices 
so if you click on the mobile's Arrow you will get a list of more mobile devices

**Reply by Alecaddd** [reply]
> If you click on the arrow beside the device name you will get a dropdown with a list of more devices
> so if you click on the mobile's Arrow you will get a list of more mobile devices

This is not a good answer Abdallah.
The input from Lucile was specifically about not having to open a dropdown and scroll through unnecessary devices:

> I don't want to have a list containing all existing devices nor only have the ones the developers chose to put in the application by default.

Let's always try to keep a welcoming environment here, especially for non-code contributors, and incorporate suggestions and requests with politeness and an open mind.

I think what Lucile listed is a great list of features for this screen, which indeed can bring the necessary flexibility and control to the users.
Great suggestions!

**Reply by LucileDT** [reply]
My bad I should have provided mockups to make my request clearer, I'll try to do that today and edit my post accordingly :slightly_smiling_face:
